bodhi/server/templates/update.html

```diff
@@ -48,6 +48,51 @@
 
     // These are the required taskotron tests
     var requirements = ${update.requirements_json | n};
+    // show the Greenwave decision
+    var greenwave_api_url = '${request.registry.settings["greenwave_api_url"]}';
+    var missing_tests = {};
+    // handle Greenwave decision
+    var handle_unsatisfied_reqs = function(data){
+        $.each(data['unsatisfied_requirements'], function(i, req) {
+            if (req.type == 'test-result-missing') {
+                if (missing_tests[req.item] === undefined)
+                    missing_tests[req.item]= [];
+                missing_tests[req.item].push(req);
+            }
+            // the user may have already specified this in the required taskotron tests
+            if ($.inArray(req.testcase, requirements) == -1) {
+                requirements.push(req.testcase);
+            }
+        });
+    };
```

Comment:
Why have the JavaScript query Greenwave when Bodhi will already know the Greenwave response? There are a few negatives to doing this:

1) Since Bodhi will poll Greenwave (at least initially until we have fedmsg), this will introduce a possibility of confusion if users see the JavaScript saying that the tests pass, but Bodhi's backend doesn't know that yet. This will lead people to ask questions in IRC. If we instead just display ```ci_status``` like we do today, it will be much less confusing for users.
2) This will put more load on the Greenwave server. ResultsDB already struggles a bit with the load that Bodhi's JS puts on it (and we've been wanting to make Bodhi cache the results there too to help with that).

I think we should avoid having the UI talk to Greenwave and just stick to using the ```ci_status``` field together with the "Automated tests" tab. I do think it would be important to display Greenwave's summary data somewhere. Perhaps @ryanlerch has ideas for the UI there?